test/core/test_integrated_tensor.py

```diff
@@ -0,0 +1,27 @@
+import pytest
```

Comment:
can you merge this with keras-core? i added the test suite #2549

Just need to move this to `tests` directory, and use the `testing` to add script related to tests which aren't tests